blog/2024-01-10-opentofu-is-going-ga.md

```diff
@@ -0,0 +1,66 @@
+Just to name an example, an [RFC for client-side state encryption](https://github.com/opentofu/opentofu/issues/874), a headline feature we wanted to have in OpenTofu, has been submitted by a community member - the same one who was trying to bring it to Terraform for years. Over the span of **multiple months** they worked to polish the PoC and RFC, involving many community members in the discussions. This RFC has recently been accepted and we’ll be collaborating with the RFC author to get it into OpenTofu 1.7. Thank you!
+
+When we were working on the registry, we had multiple RFCs submitted. We consulted numerous people all over the industry, including authors of previous similar registries like Homebrew, so that we could get the registry right on the first attempt. So far it has indeed been a success - it’s fast to `tofu init`, community members have already successfully submitted providers, and the whole thing is very cheap to run.
+
+As an open-source project, OpenTofu also benefits from sponsorships from many companies and projects. Other than the companies who started the initiative and are supporting OpenTofu with dedicated full-time engineers, we also had Cloudflare support us with hosting the registry, and BuildKite supported us with hosting release artifacts.
+
+This, and all the other discussions, issues, proposals, contributions - you name it - are the value of open-source, and what we believe will set OpenTofu apart long-term.
+
+## What’s Next?
+
+It took us a while to come out with this initial release. There has been a lot of one-time work that needed to be done to get this project set up for success. But that work is now done, and we’re ready to dive right into new development, with big new features that will set OpenTofu apart even more!
+
+First, and we know that this is important to many, we’re aiming to maintain a reasonable amount of compatibility with Terraform where it makes sense. We’re not going to be pushing for big DSL changes, we’re not going to be pushing for provider protocol changes, nothing of the sort. We’ll keep the migration path both ways easy for the foreseeable future.
+
+**The biggest change coming soon**, slated for 1.7, is the client-side state encryption. Asked for by many and for a long time now, it will let you encrypt both your state files and plan files end to end. This is valuable to projects working in a regulated environment, and ones going for maximum security. You can find the tracking issue here (todo: link).
+
+Initially, we’ll add support for user-provided keys and a couple of widely-used key management services. Long-term, depending on the usage we see and how the community responds, we might be introducing a plugin system so you can bring arbitrary key management services to it.
+
+[<img src="/img/blog/opentofu-is-going-ga-client-side-state-encryption.png" alt="A screenshot of the client-side state encryption RFC GitHub issue" width="50%"/>](https://github.com/opentofu/opentofu/issues/874)
```

Comment:
re width: This should be solved in CSS, not here. On a phone this doesn't look very good and can conflict future CSS changes. 
![image](https://github.com/opentofu/opentofu.org/assets/86970079/7c8a6e3f-a0fc-4aa8-94c5-81a05700c42e)